test/unit/builtins/numbers.spec.ts

```diff
@@ -69,6 +69,13 @@ test.each([
     util.testExpressionTemplate`(${value}).toString(2)`.expectToEqual(luaNativeSpecialNumString);
 });
 
+const toFixedRadixes = [undefined, 0, 1, 2, 5, 10, 80];
```

Comment:
Would be nice to add fractional radixes here since we support it with Math.floor